addons/medical/scripts/Game/ACE_Medical/Components/Damage/SCR_CharacterDamageManagerComponent.c

```diff
@@ -26,41 +23,50 @@ modded class SCR_CharacterDamageManagerComponent : SCR_DamageManagerComponent
 	protected bool m_bACE_Medical_SecondChanceOnHeadEnabled = false;
 	[RplProp()]
 	protected bool m_bACE_Medical_SecondChanceTriggered = false;
-	
+
 	//-----------------------------------------------------------------------------------------------------------
-	//! Initialize member variables
+	//! Initialize member variables.
 	override void OnInit(IEntity owner)
 	{
 		super.OnInit(owner);
-		
+
+		// Initialize Health Hit Zone
 		m_pACE_Medical_HealthHitZone = GetHitZoneByName("Health");
 		if (!m_pACE_Medical_HealthHitZone)
+		{
+			Print("ACE Medical: Health hit zone not found!", LogLevel.ERROR);
 			return;
-		
+		}
+
 		m_fACE_Medical_CriticalHealth = m_pACE_Medical_HealthHitZone.GetDamageStateThreshold(ECharacterHealthState.CRITICAL);
 		GetPhysicalHitZones(m_aACE_Medical_PhysicalHitZones);
+
+		// Log initialization success
+		Print("ACE Medical: Character Damage Manager initialized.", LogLevel.INFO);
 	}
-	
+
 	//-----------------------------------------------------------------------------------------------------------
 	//! Initialize ACE medical on a character damage manager (Called on the server)
 	void ACE_Medical_Initialize()
 	{
 		if (m_bACE_Medical_Initialized)
 			return;
-				
+
 		ACE_Medical_Settings settings = ACE_SettingsHelperT<ACE_Medical_Settings>.GetModSettings();
 		if (settings)
 		{
 			m_bACE_Medical_SecondChanceOnHeadEnabled = settings.m_bSecondChanceOnHeadEnabled;
 			m_fACE_Medical_SecondChanceRegenScale = settings.m_fSecondChanceRegenScale;
 		}
-		
+
 		ACE_Medical_EnableSecondChance(true);
-		// Damage calculations are done on all machines, so we have to broadcast the init
 		m_bACE_Medical_Initialized = true;
 		Replication.BumpMe();
+
+		// Log successful initialization
+		Print("ACE Medical: Initialized with second chance enabled.", LogLevel.INFO);
```

Comment:
Could you add info about `m_bSecondChanceOnHeadEnabled`, `m_fSecondChanceRegenScale` values here?